catm1_network_and_cellular_test.py

```diff
@@ -2,7 +2,6 @@
 import websocket
 import socket
 import paramiko
-import os
```

Comment:
why it removed?

Reply:
It was removed by mistake. I'll update the PR. Is there anything else that needs to be changed?

Reply:
It's added.